docs/manage/data.rst

```diff
@@ -8,27 +8,158 @@ You can download Kolibri *Detail* and *Summary* logs usage data and export in th
 	.. figure:: /img/export-usage-data.png
 	  :alt: Open Facility page, navigate to Data tab, and use the Download buttons to save the logs on your local drive. 
 
+.. note::
+  If you have more than one facility on the device, you must first select a facility. Click on the name of the facility from the list to access its data.
+
+  .. figure:: /img/select-facility.png
+    :alt: After clicking the Facility option in the sidebar, select which one you want to work on.
+
+    Select a facility to access its data.
+
+  Remember that to manage Kolibri data you must sign in as **admin** or **super admin**.
+
 To download session or summary logs, follow these steps.
 
 #. Click the *Generate log file* link for the file you require.
 #. Click the :guilabel:`DOWNLOAD` button.
-#. Save the generated ``csv`` file on your local drive.
+#. Save the generated CSV file on your local drive.
 #. To download again after more interactions click the *Generate a new log file* link.
 
-
-.. note::
-  To manage Kolibri users you must sign in as **admin** or **super admin**.
-
 .. tip::
   If the log generation is taking a long time, we recommend :ref:`using the command line interface <export_data_logs>`.
 
+.. _csv_import:
 
-Sync Facility Data
-******************
 
-.. warning::
-  This is an **experimental** feature. You can use it if you have access to the **Kolibri Data Portal**.
+Import and Export Users from a CSV file
+***************************************
+
+To import and export many users and classes at once you can use an externally generated CSV (comma separated values) file. 
+
+  .. figure:: /img/csv-import-export.png
+    :alt: 
+
+    Import and export users with CSV.
+
+.. warning:: Importing from CSV will make many changes to your users and classes, and these changes cannot be easily reverted. Read carefully the requirements below, and make sure to verify that the data in your CSV file is accurate and adheres to the format reference. 
+
+For import to be successful, you must have a CSV file with properly formatted data with some required values. For that reason we recommend the following procedure:
+
+#. Use the **Export** :ref:`feature first and save the CSV file <csv_export>` which contains all users and classes you already have in the facility.
+#. Import the CSV file in to an external spreadsheet program and make the required changes.
```

Comment:
Most programs will consider this "opening" rather than "importing"

```suggestion
#. Open the CSV file with an external spreadsheet program and make changes.
```